launcher/ui/pages/instance/ModFolderPage.h

```diff
@@ -81,3 +81,16 @@ class CoreModFolderPage : public ModFolderPage {
 
     virtual bool shouldDisplay() const override;
 };
+
+class NilModFolderPage : public ModFolderPage {
+   public:
+    explicit NilModFolderPage(BaseInstance* inst, std::shared_ptr<ModFolderModel> mods, QWidget* parent = 0);
+    virtual ~NilModFolderPage() = default;
+
+    virtual QString displayName() const override { return tr("Nilmods"); }
+    virtual QIcon icon() const override { return APPLICATION->getThemedIcon("coremods"); }
+    virtual QString id() const override { return "nilmods"; }
+    virtual QString helpPage() const override { return "Nil-mods"; }
```

Comment:
Inconsistent with displayName? I consider "Nilmods" the proper capitalization.

Reply:
That is how the help page ids are styled for loader and core mods, thought it was better to keep consistency with those (even though it does look weird)

Reply:
on second thought, the dash might be representing a space on the other two. will fix this